src/ol_infrastructure/applications/mitopen/Pulumi.applications.mitopen.Production.yaml

```diff
@@ -34,17 +41,17 @@ config:
     INDEXING_API_USERNAME: "od_mm_prod_api"
     KEYCLOAK_BASE_URL: "https://sso.ol.mit.edu"
     KEYCLOAK_REALM_NAME: "olapps"
-    MITOPEN_API_URL: "https://api.mitopen.odl.mit.edu"
-    MITOPEN_APP_URL: "https://mitopen.odl.mit.edu"
-    MITOPEN_BASE_URL: "https://mitopen.odl.mit.edu"
-    MITOPEN_COOKIE_DOMAIN: "mit.edu"
-    MITOPEN_COOKIE_NAME: "mitopen"
-    MITOPEN_LOG_LEVEL: "INFO"
-    MITOPEN_NEW_USER_LOGIN_URL: "https://mitopen.odl.mit.edu/onboarding"
-    MITOPEN_SUPPORT_EMAIL: "mitopen-support@mit.edu"
+    LEARN_API_URL: "https://api.learn.odl.mit.edu"
+    LEARN_APP_URL: "https://learn.odl.mit.edu"
+    LEARN_BASE_URL: "https://learn.odl.mit.edu"
+    LEARN_COOKIE_DOMAIN: "mit.edu"
+    LEARN_COOKIE_NAME: "learn"
+    LEARN_LOG_LEVEL: "INFO"
+    LEARN_NEW_USER_LOGIN_URL: "https://learn.odl.mit.edu/onboarding"
+    LEARN_SUPPORT_EMAIL: "learn-support@mit.edu"
     OCW_ITERATOR_CHUNK_SIZE: 300
     OIDC_ENDPOINT: "https://sso.ol.mit.edu/realms/olapps"
-    OPENSEARCH_INDEX: "mitopen"
+    OPENSEARCH_INDEX: "learn"
```

Comment:
if we change this, they will need to recreate everything in opensearch. So if we do that there is no point in retaining the existing ones and better off just recreating that infrastructure as well. 

Though maybe this rename could be worked around with a bunch of index aliases created by hand. Janky and not worth the potential risks IMO.

Reply:
Should we be using 'mitlearn' to be consistent?